components/places/src/storage/bookmarks.rs

```diff
@@ -28,7 +28,7 @@ use sync_guid::Guid as SyncGuid;
 use types::Timestamp;
 use url::Url;
 
-pub use public_node::PublicNode;
+pub use public_node::InternalBookmarkNode;
```

Comment:
The naming here is wierd, I don't think the `PublicNode` struct ever appears in a public API, does it? It seems more like an internal transition point between the database and the actual public data (previously the message type, now the uniffi dictionary type).